src/app/main/main.controller.js

```diff
@@ -8,24 +8,24 @@ angular.module('transmartBaseUi')
                 $state, StudyListService, CohortSelectionService,  GridsterService, $uibModal)
   {
 
+    var findURLQueryParams = $location.search();
```

Comment:
Why not go for a $stateParams implementation of ui-router? Its cleaner and more manageable